de.metas.endcustomer.mf15/de.metas.endcustomer.mf15.swingui/pom.xml

```diff
@@ -120,7 +120,27 @@
 			<version>${metasfresh.version}</version>
 			<scope>provided</scope>
 		</dependency>
-
+		
```

Comment:
Yes, I wasn't aware before, but that works too.
Thx to https://gist.github.com/piscisaureus/3342247.